proposals/NNNN-rc-lights-more-names-and-status-values.md

```diff
@@ -0,0 +1,75 @@
+# Remote Control – Lights – More Names and Status Values
+
+* Proposal: [SDL-NNNN](NNNN-rc-lights-more-names-and-status-values.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC]
+
+## Introduction
+
+The light status signal in the vehicle can provide more values than just ON / OFF for certain lights. Some vehicles can provide the status of more lights than currently defined light names. This proposal adds three new light status values and some new light names to allow an application to get the status of additional lights.
+
+## Motivation
+
+The light status signal in the vehicle can provide more values than just ON / OFF for certain lights. 
```

Comment:
This is already included in the `Introduction` section.  Was it intended to be repeated in the `Motivation` section?

Reply:
yes, it was inteneded

Reply:
updated